gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -134,6 +134,7 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     viewModel
         .getShowAddPolyDialogRequests()
         .observe(this, e -> e.ifUnhandled(this::onShowAddPolygonDialogRequest));
+    mapContainerViewModel.getCameraPosition().observe(this, this::checkPointNearVertice);
```

Comment:
Vertice -> Vertex

Reply:
Why in `HomeScreenFragment` and not `MapContainerFragment`? Also, I believe we already monitor camera position there.

Reply:
because all the vertex points are there in home fragment all the operation of features and observation is also handled from HomeScreenFragment itself that's the reason its done here